components/UploadedItems.js

```diff
@@ -16,7 +17,7 @@ class UploadedItems extends Component {
       .then((response) => response.json())
       .then((responseJson) => {
         const resultyt = responseJson.response.numFound;
```

Comment:
Variable names should be intuitive/self-suggestive. change it to `booksCount` or something similar. Change the state variable name as well.